spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -32,10 +32,10 @@
       it "rounds fee calculations correctly" do
         # $0.86 + 20% = $1.032
         # Fractional cents should be immediately rounded down and not carried through
-        expect(page).to have_selector '.cart-item-price',         text: '$1.03'
-        expect(page).to have_selector '.cart-item-total',         text: '$8.24'
-        expect(page).to have_selector '.order-total.item-total',  text: '$8.24'
-        expect(page).to have_selector '.order-total.grand-total', text: '$8.24'
+        expect(page).to have_selector '.cart-item-price',         text: "#{@currency_symbol}1.03"
```

Comment:
I agree... Fast and ugly workaround to skip `Capybara.default_max_wait_time = 30` when tests fail.

Unfortunately I couldn't go with `Spree::Money.new(amount).to_s` helper, because it works as full helper, meaning that it will return "$1,223.00" when 1223 is provided and in the code there and it seems that OFN codebase doesn't use `spree_number_to_currency` helper everywhere.